pkg/cli/testdata/expected_release_output.txt

```diff
@@ -98,4 +102,18 @@ from the id.
 
 #### [v0.0.5](https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.5) (2019-12-06)
 * **Added**
-    - Added ability to delete policies [#23](https://github.com/cyberark/conjur-api-python3/issues/23)
\ No newline at end of file
+    - Added ability to delete policies [#23](https://github.com/cyberark/conjur-api-python3/issues/23)
+### cyberark/conjur-api-java
```

Comment:
Oof - that header needs a newline above it but it's not a blocker since that was there already

Reply:
yeah, I saw that too ... I'm going to have other changes to the layout soon after feedback from PM, so I'd prefer to hit this then